### ISSUE TITLE
Peer review by hp444

### ISSUE THREAD

**Issue by hpuria** [SZ559/Analysis-of-Learning-Rate-for-Deep-Neural-Networks#4]
The project here aims towards analyzing the existing algorithms for finding optimal learning rates in Deep Neural Networks, study their performance and possibly try to come with a new algorithm to identify the optimal learning rate. This group also aims towards trying to improve the performance of the existing algorithms for learning rates. I didn't find the data set mentioned in the project description which would be used eventually to train/test the algorithms

Three cool things about the project:

1. The project description is very well written giving a brief introduction to deep neural networks, its application and the importance of learning rate in deep neural network. This could also help eliminate the process of setting the learning rate randomly or from past experience

2. The description clearly mentions the work/paper they would be referring for their project

3. The project has a clear timeline mentioned for each task and how the concepts being covered in class would be used in the project

Three things which could have been improved:

1. The project description doesn't mention the parameters against which the new proposed algorithm would be compared against the existing one

2. It doesn't mention the data set which would be used to validate the reliability and performance of algorithms.

3. One thing I feel which would have been interesting is if they could have given a practical example of how some processes would have performed better if a new algorithm if proposed or an improvement on an existing algorithm is suggested
